packages/design-system-tokens/src/lib/colorUtils.ts

```diff
@@ -4,6 +4,55 @@
 
 import { HexValue } from './types';
 
+export interface RgbaObject {
+  r: number;
+  g: number;
+  b: number;
+  a?: number;
+}
+
+export function rgbToHex({ r, g, b, a }: RgbaObject) {
+  if (a === undefined) {
```

Comment:
could you remove this conditional by setting a default value in the argument? `export function rgbToHex({ r, g, b, a = 1 }: RgbaObject) {}`

Reply:
Yeah, that's a good change. This came verbatim from Figma's example project